Cargo.toml

```diff
@@ -24,15 +24,15 @@ members = [
 ]
 
 [dependencies]
-libc = "~0.2"
-futures = { version = "~0.1", optional = true }
-mio = { version = "~0.6", optional = true }
+libc = "0.2.64"
+futures-preview = { version = "0.3.0-alpha.19", optional = true }
+mio = { version = "0.6.19", optional = true }
 signal-hook-registry = { version = "~1", path = "signal-hook-registry" }
-tokio-reactor = { version = "~0.1", optional = true }
+tokio-net = { version = "0.2.0-alpha.6", optional = true }
 
 [dev-dependencies]
-version-sync = "~0.8"
-tokio = "~0.1"
+version-sync = "0.8.1"
+tokio = "0.2.0-alpha.6"
```

Comment:
Why do you change the versions to not include the `~`? I like to include it because it is more readable. Without the sign, it is the same (requiring semver-compatible version), but this way it's clear it is not the *exact* version.

Furthermore, why do you increase the minimal required patch version? Do you really need at least version 0.8.1 of version-sync, etc?

Reply:
I think I ran `cargo upgrade` which seems to prefer this version format